README.md

```diff
@@ -207,80 +163,104 @@ This action uses a special directory structure, mapping subdirectory names to Se
+SensuFlow uses environment variables to customize how the script interacts with your environment.
+The minimum required variables for SensuFlow are SENSU_API_KEY and SENSU_API_URL.
+All other environment variables are optional settings that allow you to customize the SensuFlow experience.
 
-For more information, please view the SensuFlow project [issues][issues] and [milestones][milestones]. 
+Environment variable  | Type     | Required or Optional | Description
+--------------------- | -------- | -------------------- | -----------
+CONFIGURE_OPTIONS     | String   | Optional             | Additional options for the `sensuctl configure` command.
+DISABLE_SANITY_CHECKS | Boolean  | Optional             | If `true`, disable checks that ensure resource definitions are self-consistent. Otherwise, `false`. Default is `false`.
+DISABLE_TLS_VERIFY    | Boolean  | Optional             | If `true`, disable TLS certificate verification. Otherwise, `false`. Default is `false`.
+MANAGED_RESOURCES     | String   | Optional             | Comma-separated list of resource types that SensuFlow manages. Default is `"checks,handlers,filters,mutators,assets,secrets/v1.Secret,roles,role-bindings,core/v2.HookConfig"`.
+MATCHING_CONDITION    | String   | Optional             | Condition statement for a Sensu resource label match. Default is `"== 'sensu-flow'"`.
+MATCHING_LABEL        | String   | Optional             | Sensu resource label to match. Default is `"sensu.io/workflow"`.
+NAMESPACES_DIR        | String   | Optional             | Directory that SensuFlow should process. Default is `".sensu/namespaces"`.
+NAMESPACES_FILE       | String   | Optional             | Location of YAML file that contains Sensu namespace resources that SensuFlow should create. Default is `".sensu/cluster/namespaces.yml"`.
+SENSU_API_KEY         | String   | Required             | Sensu [API key][9].
+SENSU_API_URL         | String   | Required             | Sensu backend API URL (the URL used to configure `sensuctl`, such as `https://sensu.example.com:8080`).
+SENSU_CA              | String   | Optional             | Certificate Authority (CA) file as a string. Use the SENSU_CA variable if you want to encode the CA file as a GitHub Actions secret.
+SENSU_CA_FILE         | String   | Optional             | Certificate Authority (CA) file location. The SENSU_CA_FILE value (if provided) overrides the SENSU_CA value.
+VERBOSE               | Integer  | Optional             | If set, shows verbose description of actions carried out by the script. 
 
-[issues]: https://github.com/sensu/sensu-flow/issues 
-[milestones]: https://github.com/sensu/sensu-flow/milestones
+There are also two deprecated environment variables:
 
+- SENSU_PASSWORD (string): The Sensu user password.
+- SENSU_USER (string): The Sensu user to authenticate.
+
+The SENSU_API_KEY environment variable replaces the SENSU_PASSWORD and SENSU_USER variables.
+
+### Set environment variables
+
+**TODO**
```

Comment:
Are env vars for SensuFlow set differently than env vars for Sensu? -- or does this depend on the CI/CD service in use?